packages/hs-react-ui/src/components/Button/Button.tsx

```diff
@@ -75,8 +87,13 @@ export const ButtonContainer: string & StyledComponentBase<any, {}, ButtonContai
   ButtonElement,
 )`
   ${({ elevation = 0, color, variant }: ButtonContainerProps) => {
-    const backgroundColor = getBackgroundColorFromVariant(variant, color);
-    const fontColor = getFontColorFromVariant(variant, color);
+    const { transparent, background, grayDark } = useColors([
+      'transparent',
+      'background',
+      'grayDark',
+    ]);
```

Comment:
making this an array is a bit confusing to me, and they wouldn't be able to get intellisense stuff to define what color array should be passed right?

Reply:
are the keys being passed in just like requests for those items? But then we're destructuring so I feel like we could just have useColors return the entire colorscheme object

Reply:
We do get intellisense from the typescript overloads (I pulled out all the stops for this one), but as I was working I realized that you could just destructure the colors object. So, I'll just remove the other options and go with the destructure option across the board.